examples/lira_attack_example.py

```diff
@@ -129,7 +135,9 @@
     "python -m aisdc.attacks.likelihood_attack run-attack "
     "--json-file config.json "
     "--report-name example_lira_report "
-    "--n-shadow-models 100"
+    "--n-shadow-models 100 "
+    # "--shadow-models-fail-fast True "
+    # "--n-shadow-rows-confidences-min 5 "
```

Comment:
still too low probably

Reply:
@simonrnss 

Should we say 10 to be more sensible as a default value?